port/unique-id.js

```diff
@@ -0,0 +1,10 @@
+window.D2L = window.D2L || {};
+window.D2L.UniqueIFrameId = window.D2L.UniqueIFrameId || {};
```

Comment:
ESMs are scoped, so I _think_ you don't even need the globals here? Everything that imports this should get the same instance, so you could just have a local `count` variable here, initialize it to zero and then increment it. We maybe went overkill with the globals in core's version of this.

Reply:
Oh yeah, that makes sense. I'll give this a shot. Easy enough to test this in the hello-world FRA locally. :)